### ISSUE TITLE
webpack --watch does not check if stdin was closed

### ISSUE THREAD

**Issue by matthewlehner** [webpack/webpack#1308]
This is basically a clone of brunch/brunch#920, but for context:

I would like to automatically run `webpack --watch` as part of my application but, when it shuts down, the webpack process persists and continues running. I have observed that my application does close STDIN, however, `webpack --watch` is not checking if STDIN was closed or not. These two lines solve the issue:

```javascript
process.stdin.on('end', function() { process.exit(0) });
process.stdin.resume();
```

See matthewlehner/webpack@879ab0b3c4d554f68ccd17478a80a5860a33cbb8 for the fix, implemented.

I can send a pull request for this, but am wondering if this should always run this or if it should be it opt-in.

**Comment by sokra** [comment]
No sure either. Sounds like a valid PR.

**Comment by bebraw** [comment]
Is this relevant still? I can see a related PR was merged.

**Comment by matthewlehner** [comment]
Ah, this was closed with #1311
